src/Microsoft.DocAsCode.Build.Engine/PostProcessors/ExtractSearchIndex.cs

```diff
@@ -101,6 +101,9 @@ internal SearchIndexItem ExtractItem(HtmlDocument html, string href)
         {
             var contentBuilder = new StringBuilder();
 
+            if (html.DocumentNode.SelectNodes("//meta[contains(@content,'noindex')]") != null)
```

Comment:
/html/head/meta[@name="search"]?

---
In reply to: [134167442](https://github.com/dotnet/docfx/pull/1983#discussion_r134167442) [](ancestors = 134167442)